client/src/components/Workflow/Editor/Node.vue

```diff
@@ -219,6 +241,23 @@ const style = computed(() => {
     return { top: props.step.position!.top + "px", left: props.step.position!.left + "px" };
 });
 const errors = computed(() => props.step.errors || stateStore.getStepLoadingState(props.id)?.error);
+const headerClass = computed(() => {
+    let cls;
+    if (props.isInvocation) {
+        cls = "cursor-pointer";
+        if (invocationStep.value.headerClass) {
+            cls += ` ${invocationStep.value.headerClass}`;
+        } else {
+            cls += " node-header";
+        }
+    } else {
+        cls = "node-header";
+        if (!props.readonly && !props.isInvocation) {
+            cls += " cursor-move";
+        }
+    }
+    return cls;
+});
```

Comment:
defining computed classes as a `Record<string, number>` has the advantage of vastly simplifying code like this.

This change also requires changes to:
- useInvocationGraph.ts line 198
- useInvocationGraph.ts line 38

```suggestion
const headerClass = computed(() => {
    return {
        ...invocationStep.value.headerClass,
        "cursor-pointer": props.isInvocation,
        "node-header": !props.isInvocation || invocationStep.value.headerClass === undefined,
        "cursor-move": !props.readonly && !props.isInvocation,
    };
});
```

Reply:
Yes, that is better, though don't you mean `Record<string, boolean>` instead of `Record<string, number>`?